src/aat/resources/features/F-1015 - Start Case Event TTL/F-1015.feature

```diff
@@ -395,11 +486,497 @@ Feature: F-1015: Update Case - Start Case Event - Update Code for TTL
+      And a successful call [to set TTL properties for a case] as in [UpdateCase_TTLCaseType_manageCaseTTL_SuspenedNo_PreRequisiteCaseworker]
+
+     When a request is prepared with appropriate values
+      And the request [contains correctly configured event details]
+      And the request [has a TTLIncrement of 20 days configured]
+      And the request [is configured to trigger an About To Start callback that changes the TTL.OverrideTTL value (null -> missing)]
+      And the request [is configured to trigger an About To Start callback that changes the TTL.Suspended value (No -> NO)]
+      And it is submitted to call the [Start event creation process to update a case (v1_ext caseworker)] operation of [CCD Data Store]
+
+     Then a positive response is received
+      And the response has all other details as expected
+      And the response [contains the TTL.SystemTTL for the case, that has been set to 20 days from today]
+      And the response [does not contain the TTL.OverrideTTL as removed by callback (null -> missing)]
+      And the response [contains the adjusted TTL.Suspended from the callback (No -> NO)]
+
+  @S-1015.33 #CCD-3535
+  Scenario: TTLIncrement is set to "20" for the Case Event, TTL.OverrideTTL has changed (Yes -> YES) after About to Start Callback. Start Event is invoked on v1_external#/case-details-endpoint/startEventForCaseworkerUsingGET
+    Given a user with [a caseworker with an active profile in CCD]
+      And a user with [access to manage TTL properties]
+      And a successful call [to create a case] as in [CreateCase_TTLCaseType_PreRequisiteCaseworker]
+      And a successful call [to grant access to a case] as in [GrantAccess_TTLCaseType_manageTTLUser_PreRequisiteCaseworker]
+      And a successful call [to set TTL properties for a case] as in [UpdateCase_TTLCaseType_manageCaseTTL_SuspenedYes_PreRequisiteCaseworker]
+
+     When a request is prepared with appropriate values
+      And the request [contains correctly configured event details]
+      And the request [has a TTLIncrement of 20 days configured]
+      And the request [is configured to trigger an About To Start callback that changes the TTL.OverrideTTL value (null -> missing)]
+      And the request [is configured to trigger an About To Start callback that changes the TTL.Suspended value (Yes -> YES)]
+      And it is submitted to call the [Start event creation process to update a case (v1_ext caseworker)] operation of [CCD Data Store]
+
+     Then a positive response is received
+      And the response has all other details as expected
+      And the response [contains the TTL.SystemTTL for the case, that has been set to 20 days from today]
+      And the response [does not contain the TTL.OverrideTTL as removed by callback (null -> missing)]
+      And the response [contains the adjusted TTL.Suspended from the callback (Yes -> YES)]
+
+
+  @S-1015.35 #CCD-3572
```

Comment:
Tag should be #3562

Reply:
Resolved in d89cd500bb25a3455e03fcbc3f01841bb17179af